src/boot/ksdk1.1.0/devAS7262.c

```diff
@@ -54,6 +57,9 @@
 extern volatile WarpI2CDeviceState	deviceAS7262State;
 extern volatile uint32_t		gWarpI2cBaudRateKbps;
 
+//Public functions for toggling LED
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md